client/cmd/login.go

```diff
@@ -42,6 +42,10 @@ var loginCmd = &cobra.Command{
 	Short: "Authenticate at Hoop",
 	Long:  `Login to gain access to hoop usage.`,
 	Run: func(cmd *cobra.Command, args []string) {
+		if skipTLSVerifyFlag {
+			_ = os.Setenv("SKIP_TLS_VERIFY", "true")
+		}
```

Comment:
If you still want to introduce this option, make sure to include in the proper method / package. See the package `client/config/config.go`.

It should be aligned with our current way of configuring the client which consists in:
- Load this option via environment variable if all variables are set
- Load from the config toml file located at: `$HOME/.hoop/config.toml`
- [optional] Add condition to load from a local environment (for dev purporses)